scripts/document.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+if [ -z `which jazzy` ]; then
+    echo "Installing jazzy…"
+    gem install jazzy
+    if [ -z `which jazzy` ]; then
+        echo "Unable to install jazzy."
+        exit 1
+    fi
+fi
+
+
+OUTPUT=${OUTPUT:-documentation}
+
+BRANCH=$( git describe --tags --match=v*.*.* --abbrev=0 )
+SHORT_VERSION=$( echo ${BRANCH} | sed 's/^v//' )
+RELEASE_VERSION=$( echo ${SHORT_VERSION} | sed -e 's/-.*//' )
+MINOR_VERSION=$( echo ${SHORT_VERSION} | grep -Eo '^\d+\.\d+' )
+
+rm -rf ${OUTPUT}
+mkdir -p ${OUTPUT}
+
+#cp -r docs/img "${OUTPUT}"
+
+jazzy \
+    --config docs/jazzy.yml \
+    --sdk macosx \
```

Comment:
As far as I can tell, jazzy doesn’t support compiling a single docset across multiple operating systems, so we have to choose one when building the module. Fortunately, Turf has very little variation between operating systems, so I’ve chosen macOS for simplicity. The only API-level differences are on Linux, but type aliases keep things pretty consistent.